kornia/filters/unsharp.py

```diff
@@ -0,0 +1,79 @@
+from typing import Tuple
+import numpy as np
+import torch
+import torch.nn as nn
+
+import kornia
+from kornia.filters import GaussianBlur2d
+
+def sharpen(
+        input: np.ndarray,
```

Comment:
input/output must be tensors. Take into account that with the current code, you cannot use cuda.